### DISCUSSION TITLE
package does not exist in upgrade to quarkus 3.9.4

### DISCUSSION THREAD

**Discussion by Casillas0905** [quarkusio/quarkus#40210] [Q&A] [answered]
I'm trying to update the quarkus in my java app from 3.2.10.Final to the 3.9.4, but i have a problem when updating, it says this: package org.graalvm.home does not exist, the problem is that when updating quarkus this package does not longer exists and it does not said how to solved it, i tried adding this to the pom file but nothing:
(this go as dependencies)
        <dependency>
            <groupId>org.graalvm.sdk</groupId>
            <artifactId>nativeimage</artifactId>
            <version>23.1.2</version>
        </dependency>
        <dependency>
            <groupId>org.graalvm.sdk</groupId>
            <artifactId>graal-sdk</artifactId>
            <version>23.1.2</version>
        </dependency>
I had this import before the update: import org.graalvm.home.Version; to use this method: serviceType = "Quarkus " + Version.getCurrent(); , but when updating it says the package does not exists.
Any ideas why this could happen and how to solve it?
Thanks in advance

**Comment by gsmet** [comment, marked as answer]
`org.graalvm.home.Version` will return the version of GraalVM, not the version of Quarkus.

I don't think that's what you were looking for.

This SO answer explains various options on how to get your application version (`quarkus.application.version`) and the Quarkus version (`quarkus.version`): https://stackoverflow.com/questions/58306053/how-can-i-get-the-version-of-the-application-defined-by-tag-version-in-pom-xml

**Reply by Emanuel-Rad** [reply]
Hi @gsmet,

We are actually looking for 3 versions to output to our monitoring solution:
- our application version (we got that figured out)
- Quarkus version
- GraalVM version

Do you know how to get the last 2?

For GraalVM version, we used to use `import org.graalvm.home.Version;`, but that no longer works (unless we add `org.graalvm.polyglot.polyglot` as a new dependency, which we'd like to avoid)
~~And for Quarkus version, we haven't found a solution yet.~~ (update: I'll try the link you provided above)
